### ISSUE TITLE
Show next question button only after answering the question

### ISSUE THREAD

**Issue by arkaitzgarro** [arkaitzgarro/browser-quiz-project-starter#5]
We want the user to always answer the question. The next question button is hidden until the user selects an answer.

**Comment by AsiyeGokalp** [comment]
A function is created that if the answer is correct or not.
2 seconds after choosing an answer, the next question button appears if  it is not the last question

**Comment by arkaitzgarro** [comment]
> A function is created to check whether the answer is correct or not.

This will be implemented as a different task, as defined in https://github.com/arkaitzgarro/browser-quiz-project-starter/issues/4

> 2 seconds after choosing an answer, the next question button appears if it is not the last question

And this will happen in a separate PR as well, defined by this task https://github.com/arkaitzgarro/browser-quiz-project-starter/issues/8

As mentioned yesterday, I know that it may feel overkilling to have three separate PRs when it can be done in one, but it's easier to review and understand the changes.
